include/tic80_config.h

```diff
@@ -36,7 +35,6 @@
 #   define TIC_BUILD_WITH_MOON     1
 #   define TIC_BUILD_WITH_FENNEL   1
 #   define TIC_BUILD_WITH_JS       1
-#   define TIC_BUILD_WITH_WREN     1
```

Comment:
Please do not change the defaults.  By default we build with all languges.

Reply:
I now understand why you misunderstood the purpose of this PR. 
THings defined in tic80_config are not the default, but brutal forced value (imho). 
Default should and is defined in CMakeList.txt. see line 232 of the changed file. 
`set(BUILD_WITH_WREN_DEFAULT TRUE)`

If user don't intentionally disable it, it builds the same binary as current HEAD.

Reply:
I agree, pls don't change these definitions. Alternatively, you can move all of them to the cmake.

Reply:
@nesbox what do you mean by it? I did move the TIC_BUILD_WITH_WREN to Cmake and it is now controlled by  BUILD_WITH_WREN_DEFAULT whose default value is true. This is align with Janet and ruby implementation.

The PR only move one (wren) to cmake. I do have plan to move all of them but before that I need to see if you guys agree with this approach or not, thus I sent out this change for feedback.